src/services/api.js

```diff
@@ -30,4 +30,11 @@ export const createVaccinationSlot = (date) => {
     return api.post('/doctor/vaccination-slots', {
         date: date
     });
+}
+
+export const reportBug = (name, description) => {
+    return api.post(`/bugs`, {
+        name: name,
```

Comment:
If u have same key and value name in the object you can use something like this:
`{ name, description }` and js will automatically convert it to: `{ name: name, description: description }` under the hood.